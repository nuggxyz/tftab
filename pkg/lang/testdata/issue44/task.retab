working abc {
    path = "my/dir/abc/file.yaml"
}

working def {
    path = "your/dir/file.yaml"
}

working ghi {
    path = "your/dir/file-no.yaml"
}

all_file_yamls = allof("working", { key= ["path"], regex = "^.*file.yaml$" })

gen out {
    path = "out/file.yaml"
    data = { afy = [for x in all_file_yamls: x.path] }
}
