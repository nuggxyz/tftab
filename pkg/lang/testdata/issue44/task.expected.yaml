# code generated by retab v0.0.0-unknown. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "task.retab"

afy:
    - my/dir/abc/file.yaml
    - your/dir/file.yaml
